packages/react/src/components/dialog/dialogHeader/DialogHeader.module.scss

```diff
@@ -6,10 +6,11 @@
   @include dialogComponentSidePaddings;
   display: block;
   padding-top: var(--spacing-m);
-  padding-bottom: var(--spacing-3-xs);
+  padding-bottom: 0;
 }
 
 .dialogHeaderContent {
+  height: 36px;
```

Comment:
Probably it would be more failsafe to use min-height here, in case of heading line breaks.

Reply:
Good find 👍 

- [ ] use min-height instead of height